lib/components/topbar/topbar.less

```diff
@@ -135,7 +136,7 @@
     & &--container {
         align-items: center;
         display: flex;
-        height: 100%;
+        height: var(--theme-topbar-height, calc(var(--su-static48) + var(--su-static8)));
```

Comment:
@dancormier Thanks to the visual test suite I spotted that we have quite a few regression with regard to the height. While before the height was consistently applied to the whole topbar now it is applied only to the container but borders are still applied to the topbar. I played around a bit with removing the border top and the border bottom with a calc for this property but I could not figure out a solution. This is because custom properties are specifiing the whole border property and not just the border size (the number) and calculations don't work properly.

Any thoughts?

Reply:
Addressed by [this change](https://github.com/StackExchange/Stacks/pull/1692/commits/1bddec2208fe29f498b3d28aa1b7c96c339f1750). Thanks @dancormier